signac/common/config.py

```diff
@@ -12,18 +12,19 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_FILENAME = ".signacrc"
-CONFIG_FILENAMES = [DEFAULT_FILENAME, "signac.rc"]
-HOME = os.path.expanduser("~")
-CONFIG_PATH = [HOME]
-FN_CONFIG = os.path.expanduser("~/.signacrc")
+# TODO: Should these variables be internal or do we want them exposed?
```

Comment:
Since this should be documented behavior anyways I see no issue with exposing them.